modules/preview-react/menu/spec/Menu.spec.tsx

```diff
@@ -325,25 +325,25 @@ describe('Menu Keyboard Shortcuts', () => {
   it('should call the "onClose" event when the space key is pressed', () => {
     render(
       <Menu onClose={cb}>
-        <MenuItem>Alpha</MenuItem>
+        <MenuItem onClick={cb}>Alpha</MenuItem>
```

Comment:
I'm going to think about this a little bit and execute myself. I'm wondering if it already called twice and we're being more specific, or if this is to account for a change in behavior.

Reply:
better test incoming